### PR TITLE
increase size of `load all completed tasks` button

### LINKED ISSUE #1667: Increase the `Load all completed tasks` button and add an icon?
`Load all completed tasks` should have a minimal clickable size of 44px like other components and an icon, too.

See https://github.com/nextcloud/tasks/issues/1665 for further details.

### PR DESCRIPTION
**Pull request by szaimen**
Fix #1667
Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>For my own testing</summary>

```
docker run -d \
-e TASKS_BRANCH=enh/1667/increase-button-size \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.178.34 \
--name nextcloud-easy-test \
--user=www-data \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

File: src/components/DeleteCompletedModal.vue [hunk 1]
```diff
@@ -144,16 +144,16 @@ export default {
 
 <style lang="scss" scoped>
 .loadmore {
-	font-size: 11px;
-	margin-top: 10px;
+	font-size: 100%;
```
Comment (raimund-schluessler) [inline src/components/DeleteCompletedModal.vue:147]:
```suggestion
```

File: src/components/LoadCompletedButton.vue [hunk 2]
```diff
@@ -59,16 +59,16 @@ export default {
 
 <style lang="scss" scoped>
 .loadmore {
-	font-size: 11px;
-	margin-top: 10px;
+	font-size: 100%;
```
Comment (raimund-schluessler) [inline src/components/LoadCompletedButton.vue:62]:
```suggestion
```

File: src/components/DeleteCompletedModal.vue [hunk 3]
```diff
@@ -144,16 +144,16 @@ export default {
...
 	text-align: center;
-	height: 21px;
+	top: 10px;
+	position:relative;
```
Comment (raimund-schluessler) [inline src/components/DeleteCompletedModal.vue:150]:
```suggestion
	position: relative;
```

File: src/components/LoadCompletedButton.vue [hunk 4]
```diff
@@ -59,16 +59,16 @@ export default {
...
 	text-align: center;
-	height: 21px;
+	top: 10px;
+	position:relative;
```
Comment (raimund-schluessler) [inline src/components/LoadCompletedButton.vue:65]:
```suggestion
	position: relative;
```

File: src/components/DeleteCompletedModal.vue (hunk 1 above)
Comment (szaimen) [reply src/components/DeleteCompletedModal.vue]: done

File: src/components/LoadCompletedButton.vue (hunk 2 above)
Comment (szaimen) [reply src/components/LoadCompletedButton.vue]: done

File: src/components/DeleteCompletedModal.vue (hunk 3 above)
Comment (szaimen) [reply src/components/DeleteCompletedModal.vue]: done

File: src/components/LoadCompletedButton.vue (hunk 4 above)
Comment (szaimen) [reply src/components/LoadCompletedButton.vue]: done

**Comment by szaimen** [comment]
I think this should be ready now :)

**Pull request merged**